rust/private/rustc.bzl

```diff
@@ -492,6 +497,13 @@ def construct_arguments(
     # Set the SYSROOT to the directory of the rust_lib files passed to the toolchain
     env["SYSROOT"] = paths.dirname(toolchain.rust_lib.files.to_list()[0].short_path)
 
+    # extra_rustc_flags are meant to apply across all code being compiled in the target
+    # configuration, not the exec configuration.
+    # TODO(djmarcin): Should these flags apply to cfg=exec targets as well? Or via a
+    # different flag. Also, is there a better way to determine cfg=exec?
+    if hasattr(ctx.attr, "_extra_rustc_flags") and ctx.genfiles_dir.path.find("-exec-") == -1:
```

Comment:
Can you put this in a utility function in `//rust/private/utils.bzl` so it stands out a bit more?

Reply:
Done

Reply:
Thanks!